wire-ios/Wire-iOS/Sources/Authentication/Interface/Views/ValidatedTextField.swift

```diff
@@ -335,9 +339,12 @@ final class ValidatedTextField: AccessoryTextField, TextContainer, Themeable {
         validateInput()
     }
 
-    func validateInput() {
-        let error = textFieldValidator.validate(text: text, kind: kind)
+    @discardableResult
+    func validateInput(useGuestLinkRuleset: Bool = false) -> Bool {
```

Comment:
The name of the method sounds like an action and it is not clear whether it returns any value.

Reply:
Done!